test/harvest.test.js

```diff
@@ -0,0 +1,24 @@
+import fetchMock from 'fetch-mock';
+
+import { API_BASE_URL, getHarvestJSON } from '../src/lib/harvest';
+
+describe('harvest', () => {
+  test('gets JSON data from Harvest via XHR', async () => {
+    const trello = {
+      get: (scope, visibility, key) => `${scope + visibility + key}TestValue`,
+      loadSecret: (key) => `${key}TestSecret`,
+    };
+    const url = `${API_BASE_URL}some/path`;
+    fetchMock.get(url, { some: 'data' });
+    const data = await getHarvestJSON(trello, 'some/path');
+    expect(data).toEqual({ some: 'data' });
```

Comment:
Totally minor and entirely personal preference, but I find tests a bit easier to read if the assertions are separated by empty lines. 🤷‍♂ 

```suggestion

    expect(data).toEqual({ some: 'data' });
```